contrib/babelfishpg_tsql/src/guc.c

```diff
@@ -1127,6 +1128,17 @@ define_custom_variables(void)
 							 PGC_USERSET,
 							 GUC_NOT_IN_SAMPLE | GUC_DISALLOW_IN_FILE | GUC_DISALLOW_IN_AUTO_FILE,
 							 NULL, NULL, NULL);
+	/*
+	 * Block ALTER .. OWNER .. from PG endpoint
+	 */
+	DefineCustomBoolVariable("babelfishpg_tsql.enable_alter_owner_from_pg",
+							 gettext_noop("Enables blocked ALTER .. OWNER .. statements from PG endpoint"),
```

Comment:
nit: seems a bit confusing, we should rather say: `"Allows ALTER .. OWNER .. statements on Babelfish objects from PostgreSQL endpoint"`

Reply:
I think you missed to update this?